src/home/components/PetWeightPage/index.scss

```diff
@@ -0,0 +1,114 @@
+@use '~styles/colors.scss' as colors;
+@use '~styles/breakpoints.scss' as breakpoints;
+@use '~styles/fonts.scss' as fonts;
+
+.pet-weight-page {
+  min-height: 100%;
+
+  display: grid;
+
+  place-items: center;
+
+  &__content {
+    width: 100%;
+
+    text-align: center;
+  }
+
+  &__image {
+    width: max-content;
+
+    display: inline-block;
+
+    margin-bottom: 4rem;
+  }
+
+  &__description {
+    font-family: 'Montserrat', sans-serif;
+  }
+
+  &__title {
+    color: colors.$gray800;
+    font-size: fonts.$xs;
+    font-weight: fonts.$medium;
+  }
+
+  &__hint {
+    color: colors.$gray800;
+    font-size: 1.1rem;
+    font-weight: fonts.$regular;
+
+    margin-top: 1.6rem;
+  }
+
+  &__slider-container {
+    max-width: 100%;
+
+    margin-top: 3rem;
+  }
+
+  &__inputs {
+    display: flex;
+
+    align-items: center;
+    justify-content: center;
+
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+  }
+
+  &__input {
+    display: none;
+  }
+
+  &__button {
+    width: 80vw;
```

Comment:
Usar `%` pois vc quer sempre que o botão tenha `80%` to tamanho da página, não da tela